### DISCUSSION TITLE
Possible to analyze simple .csv data with asammdfgui?

### DISCUSSION THREAD

**Discussion by jt9001** [danielhrisca/asammdf#763] [General]
Hi,

I have a very simple form of a CAN data trace in a csv file, multiple lines of 

time,CAN,id,flags,dlc,data0,data1,data2,data3,data4,data5,data6,data7,counter
0.12317,1,268981664,4,8,c0,3a,08,64,00,00,01,40,1

I'd like a simple way to plot my data values, although one value for example is the combination of data2+data3.

When I load it into asammdf I don't even get the bus logging tab and an opportunity to load a .dbc file.
Because of my CAN hardware I don't see an easy way to export an mdf type file.  There is an option to export "selected signals in mdf format for Vector CANalyzer" - should I try that?

Any advice here?
Thanks!

**Comment by danielhrisca** [comment]
> Hi,
> 
> I have a very simple form of a CAN data trace in a csv file, multiple lines of
> 
> time,CAN,id,flags,dlc,data0,data1,data2,data3,data4,data5,data6,data7,counter 0.12317,1,268981664,4,8,c0,3a,08,64,00,00,01,40,1
> 
> I'd like a simple way to plot my data values, although one value for example is the combination of data2+data3.
> 
> When I load it into asammdf I don't even get the bus logging tab and an opportunity to load a .dbc file. Because of my CAN hardware I don't see an easy way to export an mdf type file. There is an option to export "selected signals in mdf format for Vector CANalyzer" - should I try that?
> 
> Any advice here? Thanks!

For sure this would be the best way

**Reply by jt9001** [reply]
Hi Daniel, thanks for replying - and I want to say awesome software.  I haven't found a lot of options for good CAN analysis so thank you for producing & maintaining this.

You're saying 'convert to selected signals to MDF for CANalyzer' is best?  I am writing the dbc from an excel document, so I am not 100% sure if it that is working well and the mdf is correct, or I'm just not using asammdf correctly.  I will experiment with known good mdf files to learn it better.

When I open my mdf file created this way I see an empty table shown below.  This makes me believe my dbc and conversion process aren't working.  Is this the best/simplest way to view an mdf for accuracy? 
![image](https://user-images.githubusercontent.com/26214777/189193262-ec0fec51-35b7-400d-b267-f3fbdd67a1b2.png)


The dbc should handle taking the 2 bytes and scaling them to be the correct value, correct?  Not done in asammdf.
Is there no way to apply a dbc to a .csv in asammdf?

Thanks a lot for the help.
